docs/installation/introduction.rst

```diff
@@ -80,6 +188,10 @@ MetalK8s uses five different **roles**, that may be combined freely:
   In practice, this role is used in conjunction with the ``master``
   and ``etcd`` roles for bootstrapping the control plane.
 
+In the :ref:`architecture diagrams<installation-intro-architecture>` presented
+above, each box represents a role (with the ``node-role.kubernetes.io/`` prefix
+omitted).
+
 .. _node-taints:
 
 Node Taints
```

Comment:
By reading at the node-taints chapter, I am not sure to understand what taints I can set and how to set it ...

Reply:
OK, I'll expand a bit with an example or two.

Reply:
I added an example and a link to the official doc. Is that enough?